### PR TITLE
Fix loading multi weight files in _pycaffe_args

### PR DESCRIPTION
**Pull request by IsaacYangSLA**
We have to loop over each pre-trained weight file when loading those files.
This fix depends on PR#851.  I will rebase this after PR#851 is merged.

### CODE REVIEW

**Comment by lukeyeager** [comment]
> I will rebase this after PR#851 is merged.

It's merged

**Comment by IsaacYangSLA** [comment]
Thanks.  Just rebased it and force pushed to my fork.

File: digits/model/tasks/caffe_train.py [hunk 1]
```diff
@@ -864,11 +864,11 @@ def _pycaffe_args(self, gpu_id):
             gpu_script = "caffe.set_device({id});caffe.set_mode_gpu();".format(id=gpu_id)
         else:
             gpu_script = "caffe.set_mode_cpu();"
+        loading_script = ""
         if self.pretrained_model:
-            weight_files = ','.join(map(lambda x: self.path(x), self.pretrained_model.split(os.path.pathsep)))
-            loading_script = "solv.net.copy_from('{weight}');".format(weight=weight_files)
-        else:
-            loading_script = ""
+            weight_files = map(lambda x: self.path(x), self.pretrained_model.split(os.path.pathsep))
```
Comment (lukeyeager) [inline digits/model/tasks/caffe_train.py]: Should we leave the `if self.pretrained_model` statement in? What if it's `None`?

Comment (IsaacYangSLA) [inline digits/model/tasks/caffe_train.py]: I think we need that if statement.  If it's None and we try to perform map on None, it throws exception.
The loading_script is init'ed as an empty string, before the if condition.  Hence, when self.pretrained_model is None, loading_script is still "", and the final script does not include statements to load pre-trained weights.

Comment (lukeyeager) [inline digits/model/tasks/caffe_train.py]: Oh sorry, I thought you had deleted it. Nevermind!

**Comment by lukeyeager** [comment]
Looks good. Thanks!

**Pull request merged**